frontend/src/components/DefenceBox/DefenceMechanism.css

```diff
@@ -1,106 +1,164 @@
+.defence-mechanism-legend {
+	padding: 0.3em;
 }
 
-.defence-mechanism > * {
-	padding: 0.75rem;
+/* the more button to expand the details */
+.defence-mechanism-summary {
+	width: fit-content;
+	padding: 0.5rem;
+	border: 0.125rem outset var(--control-config-border);
+	border-radius: 0.25rem;
 }
 
 .defence-mechanism > summary {
-	position: relative;
 	font-size: 1rem;
 }
 
-/*
- * Modified from https://www.w3schools.com/howto/howto_css_switch.asp
-*/
+/* remove the default triangle on the summary element */
+.defence-mechanism-fieldset details > summary {
+	list-style: none;
+}
 
-/* The switch - the box around the slider */
-.defence-mechanism label.switch {
+.defence-mechanism-fieldset details > summary::-webkit-details-marker {
+	display: none;
+}
+
+/* style for when details element is expanded */
+.defence-mechanism-fieldset details[open] > summary {
+	border-style: inset;
+}
+
+/* the toggle and label combined */
+.defence-mechanism-form {
 	position: absolute;
-	top: 0.75rem;
-	right: 0.75rem;
-	width: 2rem;
-	height: 1rem;
+	right: 1em;
 }
 
-.defence-mechanism label.switch:focus-within {
-	border-radius: 0.5rem;
-	outline-style: auto;
+.defence-mechanism {
+	cursor: default;
 }
 
-/* Hide default HTML checkbox */
-.defence-mechanism label.switch > input {
-	width: 0;
-	height: 0;
-	opacity: 0;
+.defence-mechanism .info-box :first-child {
+	margin-top: 0;
 }
 
-/* The slider */
-.defence-mechanism label.switch > .slider {
-	--slider-border-width: 0.125rem;
+/* the expanded description and further controls */
+.defence-mechanism .info-box {
+	padding: 0.5em;
+	font-size: 0.875rem;
+}
 
-	position: absolute;
-	inset: 0;
-	border: var(--slider-border-width) solid var(--main-toggle-off-border-colour);
-	background-color: var(--main-toggle-off-colour);
-	cursor: pointer;
-	transition: 0.4s;
+.defence-mechanism .info-box .validation-text {
+	font-weight: 600;
+	font-size: inherit;
 }
 
-.defence-mechanism label.switch > .slider::before {
-	content: '';
+.defence-mechanism .defence-radio-buttons {
+	display: flex;
+	flex-direction: column;
+	gap: 0.5rem;
+	padding: 0.75rem 0;
+}
+
+.defence-mechanism .prompt-enclosure-configuration-area {
+	padding: 0;
+}
+
+/* adapted from: https://adrianroselli.com/2019/03/under-engineered-toggles.html */
+
+/* default checkbox hidden without removing it from the DOM */
+.toggles [type='checkbox'] {
 	position: absolute;
-	top: calc(-1 * var(--slider-border-width));
-	left: calc(-1 * var(--slider-border-width));
-	width: 1rem;
-	height: 1rem;
-	background-color: var(--main-toggle-ball-colour);
-	transition: 0.4s;
+	top: auto;
+	overflow: hidden;
+	clip: rect(1px, 1px, 1px, 1px);
+	width: 1px;
+	height: 1px;
+	white-space: nowrap;
 }
 
-.defence-mechanism label.switch > input:focus + .slider {
-	box-shadow: 0 0 0.0625rem var(--main-toggle-on-colour);
+/* the  label */
+.toggles [type='checkbox'] + label {
+	position: relative;
+	display: block;
+	max-width: fit-content;
+	padding-right: 3em;
```

Comment:
Lots of `em` units showing up. I wonder if it's worth having them be rem to be more consistent with our other css? I've not looked into if it's important that they're em yet... If we want to change to rem, then we might want to check that the values convert to nice numbers of pixels (since base font is 16px)

Reply:
the article where I found the accessible toggle button says this about ems:
"All sizing is done in ems so these will scale with the surrounding text."